src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -186,7 +198,39 @@ tests:
       disabled: true
 
     - label:
-          "Step 7a: TH1 sends a RemoveAllScenes command to DUT with the GroupID
+          "Step 7: TH3 sends a StoreScene command to DUT with the GroupID field
```

Comment:
Why does this need to be manual?

Reply:
Automating TC_S_2_5 is and TC_S_2_6 is still in the scenes project backlog:
https://github.com/project-chip/connectedhomeip/issues/22813
Since the tests have to be partially manual due to the Extension Field Sets varying from device to device, this hasn't been addressed yet.